File: 251127/T를 초과하는 연속 부분 수열/subsequence-above-t.py
n, t = map(int, input().split())
arr = list(map(int, input().split()))

count, result = 0, 0
for i in range(n):
    if i >= 1 and arr[i] > t and arr[i - 1] > t:
        count += 1
    else:
        count = 1
    result = max(count, result)

print(result)

File: 260320/T를 초과하는 연속 부분 수열/subsequence-above-t.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');

const [n, t] = input[0].split(' ').map(Number);
const arr = input[1].split(' ').map(Number);

let count = 0, answer = 0;
for (let i = 0; i < arr.length; i++) {
    if (i >= 1 && arr[i - 1] > t && arr[i] > t) {
        count += 1;
    } else if (arr[i] > t) {
        count = 1;
    } else {
        count = 0;
    }

    answer = Math.max(count, answer);
}

console.log(answer);